src/status_im/ui/screens/popover/views.cljs

```diff
@@ -188,6 +189,9 @@
                   (= :activity-center view)
                   [activity-center/view request-close]
 
+                  (= :profile-share view)
```

Comment:
modal screen and `:open-modal` should be used

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/c85f47dcf261fb4e101fdb75ff9d8b7ccb34970f

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/c85f47dcf261fb4e101fdb75ff9d8b7ccb34970f